fsinfo/src/fsinfo.c

```diff
@@ -170,18 +169,10 @@ _j4status_fsinfo_format_callback(const gchar *token, guint64 value,
  * Subfunction for section_update
  * Formats block count in foo-bytes
  */
-static inline gchar *
+static inline GVariant *
 print_disk_size(fsblkcnt_t blocks, gulong bsize)
 {
-    guint64 value = (guint64) blocks * bsize << 4;
-    const gchar PREFIXES[] = "\0KMGTP"; // 1150 Pib ought to be
-                                        // enough for anybody
-    const gchar *prefix;
-    for (prefix = PREFIXES; value >= (1 << 14); prefix++)
-        value >>= 10;
-    gchar *size = g_strdup_printf(*prefix ? "%.1f %cib" : "%.1f b",
-                                  value / 16.0, *prefix);
-    return size;
+    return g_variant_new_uint64(blocks * bsize << 4);
```

Comment:
This "<< 4" isn't for calculating size, it's, like, a fixed-point arithmetic thing.  Notice I divide back by 16 at the end of func.  So if we're just passing the value as is, it should be just "blocks * bsize".
Actually, since this inline func got reduced to a single line, maybe it should be removed altogether?

Reply:
Ok, fixed.